localstack/services/cloudwatch/cloudwatch_database_helper.py

```diff
@@ -163,10 +183,8 @@ def get_metric_data_stat(
         start_time: datetime,
         end_time: datetime,
         scan_by: str,
-    ):
+    ) -> dict:
```

Comment:
We could use a generic `Dict` type here instead of dict, since we know the typing of the keys and the values.